dotnet/samples/TelemetryExample/Program.cs

```diff
@@ -105,9 +105,9 @@ private static Kernel GetKernel(ILoggerFactory loggerFactory)
         return builder.Build();
     }
 
-    private static HandlebarsPlanner CreatePlanner(int maxTokens = 1024)
+    private static HandlebarsPlanner CreatePlanner()
     {
-        var plannerConfig = new HandlebarsPlannerOptions { MaxTokens = maxTokens };
+        var plannerConfig = new HandlebarsPlannerOptions();
```

Comment:
nit: since we're aligning everything to options in this PR

```suggestion
        var plannerOptions = new HandlebarsPlannerOptions();
```